src/test/java/com/socrata/utils/TestJacksonObjectMapperProvider.java

```diff
@@ -1,56 +1,109 @@
 package com.socrata.utils;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.socrata.model.importer.Dataset;
-import junit.framework.TestCase;
-import org.codehaus.jackson.map.ObjectMapper;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.File;
 import java.io.IOException;
+import java.text.ParseException;
 import java.text.ParsePosition;
 import java.util.Date;
 
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.assertThat;
+
 /**
  * Test the JacksonObjectMapperProvider
- *
  */
-public class TestJacksonObjectMapperProvider
-{
+public class TestJacksonObjectMapperProvider {
 
-    @Test
-    public void testMapperProvider() {
+    private static final Date base = new Date(112, 5, 20, 7, 0);
```

Comment:
So, I moved this instantiation from elsewhere in the file as opposed to introducing it; therefore, can we consider this change out of scope?

There are a lot more instances of `new Date(...)` in the source and I expect you would want to address these at the same time as part of a comprehensive upgrade of the Date / Time system used by the library. For instance, you may consider replacing Joda Time with the new Java 8 java.time package. Java 7 is very old at this point, but if that's still important, you could consider the [310 back port](https://github.com/ThreeTen/threetenbp)